woocommerce/google-listings-and-ads
# Validate PHP Architecture 64 bits (Show notice)

### Changes proposed in this Pull Request:

Closes #1849 

This PR will show an admin notice in case the PHP version is working on 32 bits.

### Screenshots:

<img width="1627" alt="Screenshot 2023-01-27 at 17 30 46" src="https://user-images.githubusercontent.com/5908855/215057545-e2c626df-6477-402f-be01-4ec93f7fd1b0.png">

### Detailed test instructions:

1. Force the conditional to be true in `src/Internal/Requirements/PHPValidator.php::L58` or use PHP with 32 bits 
2. Reload the site with GLA Installed or Install/Activate it from scratch
3. See the admin notice 
4. Force the conditional to be true in `src/Internal/Requirements/PHPValidator.php::L58` or use PHP with 64 bits 
5. Reload the site with GLA Installed or Install/Activate it from scratch
6. You don't See the admin notice 


### Additional details:

I'm unsure how to switch to 32 bits PHP. If you know, please comment bellow 



### Changelog entry

> Tweak - Show admin notice when PHP 32 bits is being used
